presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleClient.java

```diff
@@ -71,6 +74,20 @@
 public class OracleClient
         extends BaseJdbcClient
 {
+    private static final Set<String> INTERNAL_SCHEMAS = ImmutableSet.<String>builder()
+            .add("information_schema")
+            .add("apex_040000")
```

Comment:
apex are tables for [Oracle Apex (Oracle Application Express)](https://apex.oracle.com/en/)
I think we should leave this